plugins/inputs/prometheus/README.md

```diff
@@ -24,6 +24,8 @@ in Prometheus format.
   ## - prometheus.io/path: If the metrics path is not /metrics, define it with this annotation.
   ## - prometheus.io/port: If port is not 9102 use this annotation
   # monitor_kubernetes_pods = true
+  ## Only monitor one namespace
+  # monitor_kubernetes_pods_namespace = "test-namespace"
```

Comment:
We try to use the default value in the commented out option, but maybe we should show an example setting, does this look right?
```
  ## Restrict Kubernetes monitoring to one namespace.
  ##   ex: monitor_kubernetes_pods_namespace = "default"
  # monitor_kubernetes_pods_namespace = ""
```

https://github.com/influxdata/telegraf/wiki/SampleConfig#style